Source/Diagnostics/ReducedDiags/MultiReducedDiags.cpp

```diff
@@ -47,10 +47,12 @@ MultiReducedDiags::MultiReducedDiags ()
             m_multi_rd[i_rd].reset
                 ( new ParticleEnergy(m_rd_names[i_rd]));
         }
-        else if (rd_type.compare("FieldEnergy") == 0)
+        else if (rd_type.compare("Field") == 0)
```

Comment:
If both the class and the file are named `FieldReduced`, maybe the input parameter can also be named `FieldReduced` for consistency? I'm okay with either way.

Reply:
I'm also ok with either way. @MaxThevenet do you have an opinion on the matter?

Reply:
Slight preference for using `FieldReduced` everywhere, as suggested by @Yin-YinjianZhao.

Reply:
:+1: I've changed the name of the input parameter to `FieldReduced`.